### ISSUE TITLE
--login-path: host parameter is no longer used

### ISSUE THREAD

**Issue by rtucek** [dbcli/mycli#807]
Hi,

I'm extensively using `mysql_config_editor` along with `--login-path`.
I've recently updated Percona MySQL via Manjaro. After the update, I've noticed that the `host` parameter is no longer honored. That is, it's required to provide `--host xyz` explicitly, even though the login-path has the host parameter set.

For reproduction:
1) Setup test-domain for localhost
```bash
echo '127.0.0.1 test.local' | sudo tee -a /etc/hosts
```

2) Setup a login-path for testing (using password 12345):
```bash
$ mysql_config_editor set --login-path test --host test.local --port 3307 --user root --password
```

3) Setup MySQL env via docker
```bash
docker run -it --rm -p 3307:3306 -e MYSQL_ROOT_PASSWORD=12345 mysql
```

4) Try to login
```bash
$ mycli --login-path test # fails
(2003, "Can't connect to MySQL server on 'test.local' ([Errno 2] No such file or directory)")

$ mysql --login-path=test # works
$ mycli --login-path=test --host test.local # works
```

Used version are:

```bash
$ mysql_config_editor --version
mysql_config_editor  Ver 8.0.18-9 for Linux on x86_64 (Source distribution)

$ mysql --version
mysql  Ver 8.0.18-9 for Linux on x86_64 (Source distribution)

$ mycli --version
Version: 1.20.1
```

**Comment by pasenor** [comment]
Fixed in version 1.24.1

**Comment by rtucek** [comment]
Yes, I can confirm it's fixed.
Thank you!
